CITATION.cff

```diff
@@ -1,37 +1,55 @@
 cff-version: 1.2.0
 message: "If you use Nocturne, please cite the Nocturne paper as below and not the GitHub repository."
+
+authors:
+- family-names: "Vinitsky"
+  given-names: "Eugene"
+- family-names: "Lichtlé"
+  given-names: "Nathan"
+- family-names: "Yang"
+  given-names: "Xiaomeng"
+- family-names: "Amos"
+  given-names: "Brandon"
+- family-names: "Foerster"
+  given-names: "Jakob"
+
+contact:
+- email: "vinitsky.eugene@gmail.com"
+  family-names: "Vinitsky"
+  given-names: "Eugene"
+- email: "nathan.lichtle@gmail.com"
+  family-names: "Lichtlé"
+  given-names: "Nathan"
+- email: "yangxm@fb.com"
+  family-names: "Yang"
+  given-names: "Xiaomeng"
+
+tile: "Nocturne"
+version: 0.0.1
```

Comment:
1.0.0? Or are we doing 0.0.1

Reply:
Let's remove version here and added it once we create releases.